### ISSUE TITLE
Uncaught Error triggered on some initialised fetch requests

### ISSUE THREAD

**Issue by ghostfuel** [SachaCR/denock#1]
Hi! This is my first time contributing to an open source repo but, thanks for starting this! :) 

I was looking for something like Nock to start writing some of my Deno tests. Good timing!

Unfortunately I am building fetch requests with RequestInit even on GET requests, which throws the following error in Denock:
`Sorry Denock only support stringified JSON as body in RequestInit object`

Which I've tracked down to [here](https://github.com/SachaCR/denock/blob/master/src/extractMethodAndBodyFromRequestInitObject.ts#L13) as there is no body present on the initialised request.

**Comment by ghostfuel** [comment]
Code to reproduce the problem:
```
import { denock } from 'https://deno.land/x/denock/mod.ts';

denock({
    method: 'GET',
    protocol: 'https',
    host: 'jsonplaceholder.typicode.com',
    path: '/todos',
    responseBody: { test: '5' },
    replyStatus: 200,
});

const urlObject = new URL('https://jsonplaceholder.typicode.com/todos');

const response = await fetch(urlObject, {
    method: 'GET'
});

const body = await response.json();

console.log(body) // ==> { test: '5' } instead of the real response.
```

**Comment by ghostfuel** [comment]
As I can't open up a branch and PR here's something simple that fixes it for me;
```
extractMethodAndBodyFromRequestInitObject.ts [Line 13]
if (init.body && typeof init.body !== "string") {

extractMethodAndBodyFromRequestInitObject.ts [Line 25]
if (init.body) {

extractMethodAndBodyFromRequestInitObject.ts [Line 28]
}
```

Let me know if you want me to open a branch or something, cheers 👍

**Comment by SachaCR** [comment]
I'll take a look at that thanks for raising this issue 👍

**Comment by SachaCR** [comment]
I have just made a fix and tagged the 0.0.2. You should be able to intercept your calls now. Thanks for your contribution.
